app/public/js/c2sScripts.js

```diff
@@ -0,0 +1,9 @@
+(function (global) {
+  'use strict';
+  $('.viewToggle a').not('.checked').attr('href', function() {
```

Comment:
Use a local copy of jQuery as in https://github.com/nhsuk/connecting-to-services/blob/master/app/public/js/geoLocation.js#L3